manager/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -6,7 +6,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: manager
```

Comment:
Just to understand where we're at: are these files still being used? Where? 

Question applies to all changes in manager/config. I'm trying to understand if they are still used in integration tests.

Reply:
They're not being used. But it was easier for me to change them to when I was verifying that both deployment mechanisms are doing the same expected thing. I'd propose that we remove them at a later point when we're sure that everything can be removed.